calico/network-policy/encrypt-cluster-pod-traffic.mdx

```diff
@@ -216,6 +218,43 @@ The above command can be used to change other WireGuard attributes. For a list o
 
 We recommend that you review and modify the MTU used by {{prodname}} networking when WireGuard is enabled to increase network performance. Follow the instructions in the [Configure MTU to maximize network performance](../networking/configuring/mtu.mdx) guide to set the MTU to a value appropriate for your network.
 
+### Enable WireGuard Threaded NAPI for a cluster
+
+:::warning
+
+There is a [known issue](https://lore.kernel.org/netdev/CALrw=nEoT2emQ0OAYCjM1d_6Xe_kNLSZ6dhjb5FxrLFYh4kozA@mail.gmail.com/T/) with WireGuard threaded NAPI that may cause NAPI to get stuck holding the global `rtnl_mutex` when a peer is removed (most likely during a node reboot when peers are removed).
+Kernels which include [this patch](https://lore.kernel.org/netdev/20240228121000.526645-2-bigeasy@linutronix.de/) are able to recover after a node drain.
+This feature should only be considered if you have high packets per second workloads that are causing dropping packets due to a saturated `softirq` CPU core.
+
+:::
+
+WireGuard Threaded NAPI increases the maximum packets per second that a WireGuard interface can process by enabling [Threaded NAPI](https://docs.kernel.org/networking/napi.html#threaded-napi) on the WireGuard interfaces.
+
+You can read more about its performance improvements [here](https://netdevconf.info/0x18/docs/netdev-0x18-paper23-talk-paper.pdf).
+
+<Tabs>
+<TabItem label="Operator" value="Operator-3">
+
+Enable Threaded NAPI WireGuard across all the nodes using the following command.
+
+```bash
+kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"wireguardThreadingEnabled":true}}'
+```
+
+This enables Threaded NAPI for both the IPv4 and IPv6 WireGuard interfaces.
+
+</TabItem>
+<TabItem label="Manifest" value="Manifest-4">
```

Comment:
I don't think we need a separate tab for Operator and manual install. Even in manual install it's possible to use `kubectl` to patch felixConfiguration.